src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -177,15 +177,15 @@ public:
     static constexpr PriorityLevel GetPriorityLevel() { return kPriorityLevel; }
     static constexpr EventId GetEventId() { return Events::{{asUpperCamelCase name}}::Id; }
     static constexpr ClusterId GetClusterId() { return Clusters::{{asUpperCamelCase parent.name}}::Id; }
-    static constexpr bool kIsFabricScoped = {{event_is_fabric_scoped}};
+    static constexpr bool kIsFabricScoped = {{isFabricSensitive}};
```

Comment:
This seems wrong... Why is it using IsFabricSensitive  for scoped?

Reply:
Sensitive implies scoped, right?  Spec says:

> Data that is fabric-sensitive SHALL always be fabric-scoped.

But more generally this is because events can be fabric sensitive, structs can be fabric scoped, and we want a single set of template machinery to work on both, treating them as fabric scoped things.  I filed https://github.com/project-chip/connectedhomeip/pull/21210#discussion_r930138152 to check what makes the most sense for events here.